ui.frontend/src/main/webpack/site/js/searchElement.ts

```diff
@@ -18,19 +18,13 @@ export const isSearchConfig = (
   const {
     id,
     searchFieldPlaceholderText,
-    autocompleteTriggerThreshold,
-    autocompleteUrl,
```

Comment:
**Thought**

why do we remove the checks? They are still sent down to frontend, right?